_server.js

```diff
@@ -0,0 +1,158 @@
+const electron = require('electron'),
+app = electron.app,
+BrowserWindow = electron.BrowserWindow,
+ipcMain = electron.ipcMain
```

Comment:
You might even try running clang-format on your code.

```
npm install --save-dev clang-format
```

Then, in your package.json, add:
```
  "scripts": {
    "format": "ls *.js | xargs clang-format --style=file -i"
  }
```

When statements don't terminate with semicolons, you'll have some crazy formatting, so be sure to fix those up.

You should probably actually have a linter here to help with that:

```
npm install --save-dev eslint
```

Then create a file called `.eslintrc.js` with this simple default config: (you can also create this by going through some prompts if you run `./node_modules/.bin/eslint --init`
```
module.exports = {
    "extends": "google"
};
```

Then extend that scripts block with another script:
```
    "lint": "eslint *.js"
```